### ISSUE TITLE
  Build failed with an exception. error ver1.7.4

### ISSUE THREAD

**Issue by jawsspot** [webim/webim-client-sdk-cordova-plugin#10]
I get error when take build me app today. I see problem in the update webim-cordova-plugin version 1.7.4
 WebimSDK.java:572: error: incompatible types: String cannot be converted to Id
        session.getStream().rateOperator(id, note, rating, new MessageStream.RateOperatorCallback() {

**Comment by ghost** [comment]
I confirm, that compile error exists

```
WebimSDK.java:572: error: incompatible types: String cannot be converted to Id
        session.getStream().rateOperator(id, note, rating, new MessageStream.RateOperatorCallback() {
```

![image](https://user-images.githubusercontent.com/83909446/159886779-766bff0a-4280-43cd-afa8-39c7b97f12a8.png)

**Comment by webim** [comment]
Issue fixed in version 1.7.5.
